api/lib/services/csvwBuilder.spec.ts

```diff
@@ -82,6 +82,7 @@ describe('csvwBuilder', () => {
         types: [
           dataCube.Table,
         ],
+        hasType: () => true,
```

Comment:
```suggestion
        types: new TermSet([
          dataCube.Table,
          dataCube.FactTable,
        ]),
```

This will require `import TermSet from '@rdfjs/term-set'`.